packages/@aws-cdk/aws-appmesh/lib/route-spec.ts

```diff
@@ -183,6 +421,11 @@ export abstract class RouteSpec {
 }
 
 class HttpRouteSpec extends RouteSpec {
+  /**
```

Comment:
No need for the comments on these classes (they are module-private - no `export` keyword).